src/pages/AppIdDetail.svelte

```diff
@@ -0,0 +1,48 @@
+<script>
+  import { getAppIdData } from "../state/api";
+
+  import { APPLICATION_ID_DEFINITION_SCHEMA } from "../data/schemas";
+  import MetadataTable from "../components/MetadataTable.svelte";
+  import Pill from "../components/Pill.svelte";
+  import PageTitle from "../components/PageTitle.svelte";
+  import { pageTitle } from "../state/stores";
+
+  export let params;
+
+  function setTitle(appId, app) {
+    pageTitle.set(`${appId.app_id} | ${app}`);
+  }
+
+  const appIdDataPromise = getAppIdData(params.app, params.appId).then(
+    (appId) => {
+      setTitle(appId, params.app);
+      return appId;
+    }
+  );
+
+  setTitle(params.appId, params.app);
```

Comment:
Just noticing the 2 `setTitle` calls here. Do the titles change once the data is loaded?

Reply:
They do, unfortunately. :( We use an encoded version of the app id in the URL, you can see this in the bread crumbs:

![image](https://user-images.githubusercontent.com/20569/110991493-aee11980-8342-11eb-9c41-bfccfd771fe2.png)

#452 tracks this issue